conf/software/LSF.nextflow.config

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env nextflow
+
+process {
+    executor = 'lsf'
+    queue = 'standard'
+    perJobMemLimit = true
+    withLabel: job_default {
+        clusterOptions = '-C0 -M1000   -R"select[mem>1000]   rusage[mem=1000]"'
+    }
+    withLabel: job_64Gb_32C {
+        clusterOptions = '-n 32 -C0 -M64000 -R"select[mem>64000] rusage[mem=64000] span[hosts=1]"'
+    }
```

Comment:
Although not entirely linked with this PR, it may be worth starting the discussion here: should we move the memory usage as the new Codon documentation states? i.e. `'-C0 -M1000` or `-n 32 -C0 -M64000 -R"span[hosts=1]"`

Reply:
How do you mean move it? Ditch the `select...` ? I have not looked at the codon docs recently :sweat_smile: